apps/web/src/app/api/v0/cron/route.ts

```diff
@@ -0,0 +1,48 @@
+import { env } from "@/env.mjs";
+import { Tinybird, publishPingResponse } from "@openstatus/tinybird";
+
+// TODO: create one route per region
+export const preferredRegion = ["fra1"];
+export const runtime = "edge";
+
+const DEFAULT_URL =
+  process.env.NODE_ENV === "development"
+    ? "http://localhost:3000"
+    : process.env.VERCEL_URL;
+
+const tb = new Tinybird({ token: env.TINY_BIRD_API_KEY });
+
+// TODO: create a package for monitor
+async function monitor(
+  res: Response,
+  { latency, url }: { latency: number; url: string }
+) {
+  return await publishPingResponse(tb)({
+    id: "openstatus",
+    timestamp: Date.now(),
+    statusCode: res.status,
+    latency,
+    url,
+    // TODO: add region, res
+  });
+}
+
+// TODO: auth middleware for user API check (unkey)
+export async function GET(req: Request) {
+  try {
+    const { searchParams } = new URL(req.url);
+    const hasUrl = searchParams.has("url");
+    const url = hasUrl ? searchParams.get("url") : `${DEFAULT_URL}/api/v0/ping`;
+
+    const startTime = Date.now();
+    const res = await fetch(url, { cache: "no-store" });
+    const endTime = Date.now();
+    const latency = endTime - startTime;
+
+    await monitor(res, { latency, url });
```

Comment:
We allow `?url=` search params. If empty, we will use our `/api/v0/ping` as default. We should never `cache` the result and we store the latency in `ms`.